### PR TITLE
fix: update webOrigins and redirectUris in keycloak for ionic-showcase compatibility

### PR DESCRIPTION
**Pull request by evanshortiss**
The default origins set by keycloak-apb don't work when the ionic-showcase is installed on an Android or iOS device.

This PR will set them correctly.

### CODE REVIEW

File: roles/setup-services/tasks/main.yml
Comment (evanshortiss) [inline roles/setup-services/tasks/main.yml:21]: The previous condition was never satisfied for me using macOS 10.14.5 and ansible-playbook 2.7.6. It would always hit 100 attempts - this resolved it.

File: roles/setup-services/tasks/main.yml
Comment (evanshortiss) [inline roles/setup-services/tasks/main.yml:63]: Similar to previous issue, the `==` condition would never be satisfied.

File: roles/setup-services/tasks/configure-keycloak.yml
Comment (evanshortiss) [inline roles/setup-services/tasks/configure-keycloak.yml:44]: From testing on iOS and Android these are the required settings. I considered adding them to `aerogearcatalog/keycloak-apb`, but seems it is better here since it is specific to ionic-showcase.

File: roles/setup-services/tasks/configure-keycloak.yml
Comment (evanshortiss) [inline roles/setup-services/tasks/configure-keycloak.yml:40]: Was unable to get `jq` and ansible to work nicely without writing this to disk. Happy to change if this is an issue and someone knows how to pass the `public_client_original.json` without getting garbled output from `jq`

File: roles/setup-services/tasks/configure-keycloak.yml
Comment (psturc) [reply roles/setup-services/tasks/configure-keycloak.yml:44]: So is `jq` installed by default on bastion servers? Should we mention in the readme that `jq` should be installed prior running this playbook?

**Review by psturc — APPROVED** [review]
one comment, otherwise LGTM
(verified on integreatly cluster)

File: roles/setup-services/tasks/configure-keycloak.yml
Comment (evanshortiss) [reply roles/setup-services/tasks/configure-keycloak.yml:44]: @psturc yep, just verified on a PDS instance. Do you think we should add a check for it in this setup?

Comment (psturc) [reply roles/setup-services/tasks/configure-keycloak.yml:44]: Yes, having this check as a first task in setup-services role would be nice 👍

Comment (evanshortiss) [reply roles/setup-services/tasks/configure-keycloak.yml:44]: @psturc added the check for `jq` 👍

**Comment by wtrocki** [comment]
💯 Thank you so much for contribution!

**Review by psturc — APPROVED** [review]

**Comment by evanshortiss** [comment]
@wtrocki @psturc thanks! Let me know if you want me to merge, or I'll leave it to you.

**Comment by wtrocki** [comment]
I have merge rights but not an actual maintainer so I might miss some details.

ping @ziccardi

**Review by wei-lee — APPROVED** [review]

**Pull request merged**
